# Request: int8 to set target brightness value, range 0-255
bool streaming

---

# Response: bool, a Bob Status code
int8 status
